Config/SEGGER_RTT_Conf.h

```diff
@@ -98,7 +98,7 @@ Revision: $Rev: 24316 $
 #define SEGGER_RTT_SECTION                        ".dtcm_data"
 #elif defined(CONFIG_SEGGER_RTT_SECTION_CCM)
 #define SEGGER_RTT_SECTION                        ".ccm_data"
-#elif defined(CONFIG_SEGGER_RTT_SECTION_CUSTOM)
+#elif defined(CONFIG_SEGGER_RTT_SECTION_CUSTOM) || defined(CONFIG_SEGGER_RTT_SECTION_CUSTOM_DTS_REGION)

```

Comment:
Where is `CONFIG_SEGGER_RTT_SECTION_CUSTOM_DTS_REGION` declared?

Reply:
Ref see last comment https://github.com/zephyrproject-rtos/zephyr/pull/53569/files

Reply:
Correct, this PR depends on [PR #53569](https://github.com/zephyrproject-rtos/zephyr/pull/53569): there the `choice SEGGER_RTT_SECTION` in `modules/segger/Kconfig` has been extended,  `SEGGER_RTT_SECTION_CUSTOM_NAME` and `choice SEGGER_RTT_INIT_MODE` have been added.